data-protocols/dsp/dsp-negotiation/dsp-negotiation-transform/src/main/java/org.eclipse.edc.protocol.dsp.negotiation.transform/to/JsonObjectToContractAgreementVerificationMessageTransformer.java

```diff
@@ -37,8 +38,10 @@ public JsonObjectToContractAgreementVerificationMessageTransformer() {
     public @Nullable ContractAgreementVerificationMessage transform(@NotNull JsonObject object, @NotNull TransformerContext context) {
```

Comment:
Since we are renaming `object` for clarity, probably it make sense to rename also the `JsonObject` ones also in the transformers